pipelines/openjdk10_nightly_pipeline.groovy

```diff
@@ -0,0 +1,42 @@
+println "building OpenJDK10"
+stage 'build OpenJDK'
+def Platforms = [:]
+Platforms["Linux"] = {
+    node {
+        build job: 'openjdk10_build_x86-64_linux', propagate: false
```

Comment:
Is there a reason why the x86-64 build is the only one with `propagate: false`?

Reply:
I'm actually not sure, reading the docs it says that setting to false lets the next build step execute even if this one was unstable.  I can't see why we'd want that so I've removed it.

Reply:
Right so, as the code reads now, if any builds fails, except for x86-64, don't execute the `checksums` stage